extension-iap/src/iap_ios.mm

```diff
@@ -324,28 +325,25 @@ static void HandlePurchaseResult(IAPCommand* cmd)
     assert(top == lua_gettop(L));
 }
 
-@implementation SKPaymentTransactionObserver
-    - (void)paymentQueue:(SKPaymentQueue *)queue updatedTransactions:(NSArray *)transactions
-    {
-        for (SKPaymentTransaction* transaction in transactions) {
-
-            if ((!self.m_IAP->m_AutoFinishTransactions) && (transaction.transactionState == SKPaymentTransactionStatePurchased)) {
+static void processTransactions(IAP* m_IAP, NSArray* transactions) {
```

Comment:
nitpick: function arguments should be lowercase, and also not have the prefix.
Replace with `iap` here.

Reply:
@dyukorev Could you please address this?